portal/templates/patients_by_org.html

```diff
@@ -102,6 +102,76 @@ <h4 class="tnth-headline">Patient List</h4>
   </div>
   <div id="admin-table-error-message" class="text-danger smaller-text"></div>
 </div>
+{% if 'reports' in config.PATIENT_LIST_ADDL_FIELDS %}
+  <div class="modal fade" id="patientReportModal" tabindex="-1" role="dialog" aria-labelledby="patientReportModal">
+      <div class="modal-dialog" role="document">
+          <div class="modal-content">
+              <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title">{{ _("Patient Report") }}</h4>
+              </div>
+              <div class="modal-body">
+                  <div id="patientReportLoader" class="tnth-hide"></div>
+                  <div id="patientReportContent"></div>
+                  <div id="patientReportMessage" class="error-message"></div>
+              </div>
+              <div class="modal-footer" >
+                  <button type="button" class="btn btn-default btn-consent-close" data-org="{{org}}" data-dismiss="modal" aria-label="Close">{{ _("Close") }}</button>
+              </div>
+          </div>
+      </div>
+  </div>
+  <script>
```

Comment:
Would you be able to move this to `main.js` or another JS file?

Moving JS from dynamically generated HTML into javascript files can improve caching and *eventually* allow us to block inline JS, closing a lot of security vulnerabilities.

No big deal if this isn't feasible at the moment :)

Reply:
ivan, thank u for the suggestion, I have moved the block to admin.js.